### ISSUE TITLE
java.lang.IllegalArgumentException: value already present

### ISSUE THREAD

**Issue by steve4744** [IntellectualSites/PlotSquared#2423]
__*NOTICE: Bukkit/Spigot versions 1.7.10 to 1.12.2 are considered legacy and will receive limited support. Please consider upgrading to 1.13 for future support. Plugins exist for 1.13+ that bring back old behaviors found in 1.8*__
# Bug report template 

**[REQUIRED] PlotSquared Version Number:** 
v4.298
**[REQUIRED] Spigot/Paper Version Number:** 
1.14.3
**[REQUIRED] Minecraft Version Number:** 
1.14.3
**Links to worlds.yml file and settings.yml file:** 
https://pastebin.com/kJgG2z3V
https://pastebin.com/jjBmrwEn

**[REQUIRED] Description of the problem:** 
Trying to upgrade from 1.13.2 (PS v4.228) to 1.14.3 (PS v4.298)  - message logged to console.
All plots (apart from my own) are now owned by "Unknown".

The uuid listed in the console output is a player who has not been on the server since 2014.

**Any relevant console output or screenshots:**
```
[01:11:46] [Craft Scheduler Thread - 2/INFO]: [P2] Starting player data caching for: world
[01:11:46] [Craft Scheduler Thread - 2/WARN]: [PlotSquared] Plugin PlotSquared v4.298 generated an exception while executing task 69
java.lang.IllegalArgumentException: value already present: 14d602b9-2ce2-4ddd-9214-682a7a00f295
        at com.google.common.collect.HashBiMap.put(HashBiMap.java:287) ~[spigot-1.14.3.jar:git-Spigot-5e4e7f3-5f88938]
        at com.google.common.collect.HashBiMap.put(HashBiMap.java:262) ~[spigot-1.14.3.jar:git-Spigot-5e4e7f3-5f88938]
        at com.github.intellectualsites.plotsquared.bukkit.uuid.FileUUIDHandler.lambda$cache$0(FileUUIDHandler.java:220) ~[?:?]
        at org.bukkit.craftbukkit.v1_14_R1.scheduler.CraftTask.run(CraftTask.java:81) ~[spigot-1.14.3.jar:git-Spigot-5e4e7f3-5f88938]
```

**Plugins being used on the server:**

**How to replicate:** 
Start server with PS v4.298

**Checklist**:

- [] I included all information required in the sections above
- [] I made sure there are no duplicates of this report [(Use Search)](https://github.com/IntellectualSites/PlotSquared/issues?utf8=%E2%9C%93&q=is%3Aissue)
- [] I made sure I am using an up-to-date version of PlotSquared
- [] I made sure the bug/error is not caused by any other plugin

**Comment by steve4744** [comment]
Can this be re-opened as d6779d9 does not appear to have fixed the issue?
```
 [14:28:19] [Craft Scheduler Thread - 6/WARN]: [PlotSquared] Plugin PlotSquared **v4.303** generated an exception while executing task 76
java.lang.IllegalArgumentException: value already present: 14d602b9-2ce2-4ddd-9214-682a7a00f295
        at com.google.common.collect.HashBiMap.put(HashBiMap.java:287) ~[spigot-1.14.4.jar:git-Spigot-9de398a-9c887d4]
        at com.google.common.collect.HashBiMap.put(HashBiMap.java:262) ~[spigot-1.14.4.jar:git-Spigot-9de398a-9c887d4]
        at java.util.Map.putIfAbsent(Unknown Source) ~[?:1.8.0_212]
        at com.github.intellectualsites.plotsquared.bukkit.uuid.FileUUIDHandler.lambda$cache$0(FileUUIDHandler.java:221) ~[?:?]
        at org.bukkit.craftbukkit.v1_14_R1.scheduler.CraftTask.run(CraftTask.java:81) ~[spigot-1.14.4.jar:git-Spigot-9de398a-9c887d4]
        at org.bukkit.craftbukkit.v1_14_R1.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:54) [spigot-1.14.4.jar:git-Spigot-9de398a-9c887d4]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source) [?:1.8.0_212]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source) [?:1.8.0_212]
        at java.lang.Thread.run(Unknown Source) [?:1.8.0_212]
```

**Comment by MattBDev** [comment]
Looks like it could be a possible race condition

**Comment by steve4744** [comment]
To add an update to this, I removed the offending \<uuid\>.dat file from the world\playerdata folder and restarted the server. It logged the same error with a different \<uuid\> twice more. Each time I removed the \<uuid\>.dat file and restarted.

The server now starts normally and all the plots are owned by the correct players.

The 3 \<uuid\>.dat files that were removed belong to players that haven't been on the server since 2014, and didn't own any plots.
There is one \<uuid\>.dat file remaining that is date-stamped 2014. That player does own a plot but it doesn't seem to be causing the same issue.
